### PR TITLE
e2e test: create+delete create+clear in 1 tx

### PR DESCRIPTION
**Pull request by algonautshant**
create + delete in one tx should pass
create + clear in one tx should fail

### CODE REVIEW

File: test/scripts/e2e_subs/e2e-app-other.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+date '+keyreg-teal-test start %Y%m%d_%H%M%S'
```
Comment (algorandskiy) [inline test/scripts/e2e_subs/e2e-app-other.sh:3]: not a rekey test

File: test/scripts/e2e_subs/e2e-app-other.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+WALLET=$1
+
+gcmd="goal -w ${WALLET}"
+
+ACCOUNT=$(${gcmd} account list|awk '{ print $3 }')
+
+# approval program
+printf '#pragma version 2\nint 1' > "${TEMPDIR}/simple.teal"
+PROGRAM=($(${gcmd} clerk compile "${TEMPDIR}/simple.teal"))
```
Comment (algorandskiy) [inline test/scripts/e2e_subs/e2e-app-other.sh:18]: isn't too many parenthesis?

File: test/scripts/e2e_subs/e2e-app-other.sh (hunk 1 above)
Comment (algonautshant) [reply test/scripts/e2e_subs/e2e-app-other.sh:3]: Fixed. Also fixed other tests with the same error.

File: test/scripts/e2e_subs/e2e-app-other.sh (hunk 2 above)
Comment (algonautshant) [reply test/scripts/e2e_subs/e2e-app-other.sh:18]: This statement is not needed anyway. Thanks for pointing it out.

**Pull request merged**